bancor_research/scenario_generator/server.py

```diff
@@ -3,8 +3,18 @@
 # Licensed under the MIT LICENSE. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------------------------------
 """Sets up the interactive visualization server for the Mesa Agent-based simulation."""
+from decimal import Decimal
+
+import mesa
 
 # The colors here are taken from Matplotlib's tab10 palette
+from bancor_research.scenario_generator.model import BancorSimulation
+
+from bancor_research.scenario_generator.constants import SIMULATION_WHALE_THRESHOLD
+
+from bancor_research.bancor_simulator.v3.spec import DEFAULT_TRADING_FEE, DEFAULT_NETWORK_FEE, DEFAULT_WITHDRAWAL_FEE
+from bancor_research.scenario_generator.agents import Trader, LP
```

Comment:
You can import some of the values here directly from the top-level module:
```
from bancor_research import DEFAULT
```
Then replace every occurrence of `DEFAULT_` with `DEFAULT.`.

Not critical, since we have some work around duplicated default values also in other places in the code.

Reply:
done